### PR TITLE
docs(readme): add lighthouse-batch-parallel to related projects

### PR DESCRIPTION
**Pull request by Carr1005**
**Summary**

This PR adds Lighthouse [lighthouse-batch-parallel](https://github.com/Carr1005/lighthouse-batch-parallel) to the list of related projects!

### CODE REVIEW

**Comment by patrickhulce** [comment]
thanks for the PR @Carr1005 and great project! (love those docs too 😍)

lately though [we've been recommending against](https://github.com/GoogleChrome/lighthouse/issues/7104#issuecomment-458368476) running multiple LH runs in parallel unless you have particularly beefy setups, do you think we could some sort of disclaimer like that in your project since it actively promotes it? :)

**Comment by Carr1005** [comment]
@patrickhulce, thanks for reviewing! So lame that I just made a tool completely built by a not recommended way (with a very loud name 😂) I already put the disclaimer in my project, though I am not sure if this is valuable to be added (as a tool to prove how wrong it can be 😂?)

I am down for just closing this pr, maybe I'll create a new one without the feature multiple monitoring in parallel and just make another pr here at some points :)

File: readme.md [hunk 1]
```diff
@@ -381,6 +381,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lightcrawler](https://github.com/github/lightcrawler)** - Crawl a website and run each page found through Lighthouse.
 * **[lighthouse-badges](https://github.com/emazzotta/lighthouse-badges)** - Generate gh-badges (shields.io) based on Lighthouse performance.
 * **[lighthouse-batch](https://www.npmjs.com/package/lighthouse-batch)** - Run Lighthouse over a number of sites and generate a summary of their metrics/scores.
+* **[lighthouse-batch-parallel](https://www.npmjs.com/package/lighthouse-batch-parallel)** - Run multiple Lighthouse in parallel to accelerate the data collecting process, get the result stream (csv, json, js object) in your own process.
```
Comment (patrickhulce) [inline readme.md]:
```suggestion
* **[lighthouse-batch-parallel](https://www.npmjs.com/package/lighthouse-batch-parallel)** - Run multiple Lighthouse runs in parallel to accelerate the data collecting process, get the result stream (csv, json, js object) in your own process (warning: performance results may be volatile).
```

**Comment by Carr1005** [comment]
@patrickhulce, Thanks for the support! :raised_hands:

**Pull request merged**